sdk/src/services/AccountFactoryService.ts

```diff
@@ -167,6 +167,7 @@ export class AccountFactoryService {
      * @param requestBody
      * @param simulateTx Simulates the transaction before adding it to the queue, returning an error if it fails simulation. Note: This step is less performant and recommended only for debugging purposes.
      * @param xIdempotencyKey Transactions submitted with the same idempotency key will be de-duplicated. Only the last 100000 transactions are compared.
+     * @param xTransactionMode Transaction mode to use for EOA transactions. Will be ignored if using a smart wallet. If omitted, defaults to regular EOA transactions.
```

Comment:
Was thinking it could also be used to turn sponsoring on/off for smart wallets. Fine for now though